firefox-ios/Shared/Strings.swift

```diff
@@ -7574,6 +7574,29 @@ extension String {
     }
 }
 
+// MARK: - Live Activity
+extension String {
+    public struct LiveActivity {
+        public struct Downloads {
+            public static let FileNameText = MZLocalizedString(
+                key: "LiveActivity.Downloads.FileNameText.v137",
+                tableName: "LiveActivity",
+                value: "Downloading \"%@\"",
```

Comment:
Over here we should favor typographical quotes over the ascii ones as per https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#land-good-quality-strings. I believe you shouldn't have to escape this quote type.
<img width="794" alt="Screenshot 2025-02-20 at 10 30 19 AM" src="https://github.com/user-attachments/assets/94e69f1d-a707-4502-aa8a-658b2b275cc5" />